integration/ig/k8s/trace_exec_test.go

```diff
@@ -89,3 +90,64 @@ func TestTraceExec(t *testing.T) {
 
 	RunTestSteps(commands, t, WithCbBeforeCleanup(PrintLogsFn(ns)))
 }
+
+func TestTraceExecHost(t *testing.T) {
+	t.Parallel()
+
+	cmd := "sh -c 'for i in $(seq 1 30); do date ; /bin/sleep 0.1; done'"
+	shArgs := []string{"/bin/sh", "-c", cmd}
+	dateArgs := []string{"/bin/date"}
+	sleepArgs := []string{"/bin/sleep", "0.1"}
+
+	traceExecCmd := &Command{
+		Name:         "TraceExecHost",
+		Cmd:          "ig trace exec -o json --host",
+		StartAndStop: true,
+		ExpectedOutputFn: func(output string) error {
+			expectedEntries := []*execTypes.Event{
+				{
+					Comm:  "sh",
+					Args:  shArgs,
+				},
+				{
+					Comm:  "date",
+					Args:  dateArgs,
+				},
+				{
+					Comm:  "sleep",
+					Args:  sleepArgs,
+				},
+			}
+
+			normalize := func(e *execTypes.Event) {
+				// Event is running on the host, so no containers related data.
+				e.Namespace = ""
+				e.Pod = ""
+				e.Container = ""
```

Comment:
Since you expect those fields to be empty, it should not be reset to "" in the normalize function, otherwise the test would not fail if they are not empty.